### ISSUE TITLE
NVDA doesn't announce proper object type while consuming a word document with multiple object inserted.

### ISSUE THREAD

**Issue by WiproPSAccTest** [nvaccess/nvda#9394]
### Steps to reproduce:
1. Open a word document with multiple objects(Illustrations_ObjectType_Inline_TestDoc)
2. Navigate in the object with Down Arrow
3. Observe that NVDA is not announcing object types properly for different objects. 
4. For few objects NVDA announces objects as "/"(Slash) and at that time it is no reading "Alt text" also.
Shapes are announced as Slash
Textboxes are announced as Slash
Ink are announced as Slash
Online Pictures are announced as Slash

5. For few other objects NVDA announces as "Embedded Object", this time NVDA reads the alt text
Icons are announced as Embedded object
Online Video are announced as Embedded object
3D Models are announced as Embedded object
SmartArt are announced as Embedded object

### Actual behavior:
NVDA doesn't announce proper object type while consuming a word document with multiple object inserted.

### Expected behavior:
All the object announcement should be proper and expected to the user.

#### NVDA version:
2018.4.1
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA?
Yes, same everywhere
